packages/app/src/components/Live/Live.tsx

```diff
@@ -26,6 +28,13 @@ type User = {
     isBurner: boolean | null;
 };
 
+type Props = {
```

Comment:
Since we switched back to using reactNavigation, we will need to do the props the same way as before.